queries.sql

```diff
@@ -21,3 +21,59 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
+--Inside a transaction delete all records in the animals table, then roll back the transaction.
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
for the requirements to update the table and also to delete some records, I noticed you didn't add the code to verify that the action was successful as required. please kindly add the code to show that the actions you performed worked. you did this properly in your PR description but you didn't do it here. and it is more important here😉.

<img width="1068" alt="image" src="https://github.com/danifromecuador/vet-clinic/assets/64643995/9eea020b-482b-4fd3-923d-7c6cb619f020">

Reply:
Thank you @Ibizugbe I've included `SELECT * FROM animals;` inside each transaction to verify if the changes were made.